.github/workflows/matrix.py

```diff
@@ -28,7 +28,7 @@
 os_release_list = [
     'ubuntu-20.04',
     'windows-latest',
-    'macos-latest',
+    'macos-latest', 
```

Comment:
Would be nice to not load in a whitespace diff

Reply:
fixed